diff --git a/src/panel.c b/src/panel.c
--- a/src/panel.c
+++ b/src/panel.c
@@ -79,7 +79,7 @@
             "anchor", ZWLR_LAYER_SURFACE_V1_ANCHOR_BOTTOM
             | ZWLR_LAYER_SURFACE_V1_ANCHOR_LEFT
             | ZWLR_LAYER_SURFACE_V1_ANCHOR_RIGHT,
-            "layer", ZWLR_LAYER_SHELL_V1_LAYER_TOP,
+            "layer", ZWLR_LAYER_SHELL_V1_LAYER_OVERLAY,
             "kbd-interactivity", FALSE,
             "exclusive-zone", height,
             "namespace", "osk",
